api/cmd/manager/main_test.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
do we need to skip this file in unit test?

Reply:
Yes, I already config it to skip it when run unit test